gemini/sample-apps/finance-advisor-spanner/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.9-slim
```

Comment:
Should probably use the latest version of Python if possible.

Reply:
done